frontend/src/features/notes/list/NoteResults/NoteResults.test.tsx

```diff
@@ -15,11 +15,12 @@ const setup = (renderOptions: RenderOptions & Partial<INoteResultProps> = { resu
   const { results, ...rest } = renderOptions;
   const utils = render(
     <NoteResults
+      onShowDetails={jest.fn()}
       sort={{}}
       results={results ?? []}
       setSort={setSort}
       onDelete={noop}
-      onShowDetails={noop}
+      //onShowDetails={noop}
```

Comment:
looks not useful given line 18 has the same prop

Reply:
removed

Reply:
It does not look removed, but commented out